.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+
+executors:
+  node-container:
+    docker:
+      - image: node:12
+        environment:
+          NPM_CONFIG_PROGRESS: 'false'
+          NPM_CONFIG_LOGLEVEL: warn
+
+jobs:
+  dependency-check:
+    executor: node-container
+    steps:
+      - checkout
+      - run:
+          name: Create Dependency Check sandbox
+          command: mkdir -p ./dependency-check
+      - run:
+          name: Create Package JSON
+          command: node ./scripts/dependency-check/create-manifest-from-template
+      - run:
+          name: Check Outdated
+          working_directory: ./dependency-check
+          command: (npm outdated --json || true ) > ./outdated.json
+      - run:
+          name: Create new template
+          command: node ./scripts/dependency-check/create-template-from-outdated
+      - run:
+          name: Run Tests and Produce Reports
+          command: |
+            FORMER_CWD=$(pwd)
+            PROJECT=test-$(date +%s)
+            mkdir "${TMPDIR}/${PROJECT}"
+            cd "${TMPDIR}/${PROJECT}"
+            npx @contentful/create-contentful-app init test
+            cd test
+            npm t > "${FORMER_CWD}/dependency-check/test-report"
+            cat "${FORMER_CWD}/dependency-check/test-report"
+      - run:
+          name: Create Pull Request
+          command: echo 'PR' #npm install @octokit/rest simple-git && node ./scripts/dependency-check/create-pull-request
```

Comment:
If you do it like that you should install at specific version (for example `simple-git@2.5`)